index.html

```diff
@@ -110,29 +111,35 @@ <h3>how we started</h3>
             leggings scenester vape meditation offal blog thundercats semiotics chicharrones.
             Offal iPhone forage sustainable. Cronut irony chillwave poke celiac.
         </p>
-        <strong>Next level iceland blue bottle fingerstache:</strong>
-        <ul>
-          <li>hella ennui banh mi iPhone kitsch prism VHS marfa fixie</li>
-          <li>Typewriter direct trade shoreditch,small batch hexagon</li>
-          <li>Poutine jean shorts kale chips normcore.</li>
+        <strong class="team__values-title">Next level iceland blue bottle fingerstache:</strong>
```

Comment:
Może to jest osobny component? Albo może na razie nie ma potrzeby dodawać tych klas? Co tam będzie do ostylowania? Wygląda jakby to albo miał być nowy component i wtedy np. (nie upieram się że to dobra nazwa) `team-values__title`, `team-values__list`, `team-values__list-item`.

Reply:
Tak, osobny komponent, mhm. Może faktycznie nie są potrzebne :p